### PR TITLE
Overview of changes in the GAP 4.10.1 release

### PR DESCRIPTION
**Pull request by alex-konovalov**

### CODE REVIEW

**Comment by alex-konovalov** [comment]
This looks good, and I checked how the manuals builds in https://travis-ci.org/gap-system/gap/jobs/497512315. Waiting for someone to review and approve.

File: doc/changes/changes-4.10.xml [hunk 1]
```diff
@@ -623,4 +623,185 @@ fix the package, the latest sources are available at
...
+<Item>
+Remove <C>environ</C> arguments and <C>sysenviron</C>
+(<URL><LinkText>&Hash;3111</LinkText><Link>https://github.com/gap-system/gap/pull/3111</Link></URL>).
+</Item>
+</List>
+
+Fixes in the experimental support for using the Julia garbage collector:
+<List>
+<Item>
+Fix task scanning for Julia GC
```
Comment (fingolfin) [inline doc/changes/changes-4.10.xml]: "The" is missing

Comment (alex-konovalov) [reply doc/changes/changes-4.10.xml]: @fingolfin fixed. Tests passed.

**Comment by fingolfin** [comment]
This was cherry-picked to `stable-4.10` in commit 6763890b4a8c9e340d6fba12c0c9c169970945ac

**Pull request merged**